crates/kitsune_p2p/kitsune_p2p/src/types/event.rs

```diff
@@ -164,7 +165,6 @@ pub fn full_time_window_inclusive() -> TimeWindowInclusive {
 
 type KSpace = Arc<super::KitsuneSpace>;
 type KAgent = Arc<super::KitsuneAgent>;
-type KOpHash = Arc<super::KitsuneOpHash>;
```

Comment:
This type is already defined. I had made more changes here but they weren't necessary so I've removed them but this seems like a worthwhile de-duplication to keep